source/messaging/channel-preferences.rst

```diff
@@ -18,53 +18,96 @@ Set Channel Preferences
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-You can set channel preferences, such as notifications, as well as the channel's header, purpose, and name for each channel you belong to. To access these settings, select the channel name at the top of the page.
+For each channel you belong to, you can set notification preferences as well as the channel's header, purpose, and name. Select the channel name at the top of the screen to access channel-specific settings.
+
+.. image:: ../images/channel-preferences.png
+        :alt: Select the channel name at the top of the screen to set preferences for each channel you belong to.
 
 Channel notification preferences
 --------------------------------
 
-Each channel you belong to can have different notification preferences.
+Change the following channel notification settings by selecting **Notification Preferences**.
 
 Mute channel
 ~~~~~~~~~~~~~
 
-By default, channel muting is turned off for all channels. To mute or unmute a channel, select the channel name at the top of the page to access the channel menu, then choose **Mute Channel**.
+By default, all chanels are unmuted. To mute or unmute the current channel, enable the **Mute Channel** option, then select **Save**.
```

Comment:
What does muting mean?...

Muting turns off desktop, email and push notifications for this channel. You'll still see a mention badge in the sidebar if you're mentioned specifically.